### PR TITLE
Chore: e2e tests on matching component (Issue/181)

### PR DESCRIPTION
**Pull request by lemmyadams**
Addresses #181

### CODE REVIEW

**Comment by lemmyadams** [comment]
Relies on https://github.com/adaptlearning/adapt_framework/pull/3533

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]

File: test/e2e/matching.cy.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  it('should display the matching component', function () {
+    const matchingComponents = this.data.components.filter((component) => component._component === 'matching')
+    matchingComponents.forEach((matchingComponent) => {
+      cy.visit(`/#/preview/${matchingComponent._id}`);
+      const bodyWithoutHtml = cy.helpers.stripHtml(matchingComponent.body)
+      cy.testContainsOrNotExists('.matching__body', bodyWithoutHtml)
+
+      cy.testContainsOrNotExists('.matching__title', matchingComponent.displayTitle)
+      cy.testContainsOrNotExists('.matching__instruction', matchingComponent.instruction)
```
Comment (oliverfoster) [inline test/e2e/matching.cy.js:15]:
```suggestion
      const stripHtml = cy.helpers.stripHtml;
      cy.testContainsOrNotExists('.matching__body', stripHtml(bodyWithoutHtml))
      cy.testContainsOrNotExists('.matching__title', stripHtml(matchingComponent.displayTitle))
      cy.testContainsOrNotExists('.matching__instruction', stripHtml(matchingComponent.instruction))
```

File: test/e2e/matching.cy.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+describe('Matching', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the matching component', function () {
+    const matchingComponents = this.data.components.filter((component) => component._component === 'matching')
+    matchingComponents.forEach((matchingComponent) => {
```
Comment (oliverfoster) [inline test/e2e/matching.cy.js:9]:
```suggestion
    matchingComponents.forEach(matchingComponent => {
```

These brackets aren't needed, please remove them from fat-arrow functions.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax

**Review by oliverfoster — CHANGES_REQUESTED** [review]

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**